src/Analyzers/VisualBasic/Tests/UseNullPropagation/UseNullPropagationTests.vb

```diff
@@ -1104,5 +1104,20 @@ public class C
     end sub
 end class")
         End Function
+
+        <Fact>
+        Public Async Function TestElseIf() As Task
+            ' Subject to improve
+            Await TestMissingInRegularAndScriptAsync(
+"
+Class C
+    Sub M(s as String)
+        If True Then
+        ElseIf s [||]IsNot Nothing
+            s.ToString()
+        End If
+    End Sub
+End Class")
```

Comment:
Add test with a true nested if statement within the else. Again, just to make sure no crashes

Reply:
I might be getting a bit pesky, but can you clarify here as well? You want something like that or what:
```vb
...
ElseIf x IsNot Nothing
    If True
        x.ToString()
    End If
End If
```

Reply:
No elseif :-).  I def recognize we won't do any special. I just want the coverage.